src/deploy/RPM_build/noobaa.spec

```diff
@@ -1,62 +1,88 @@
 %define buildroot %{_tmppath}/%{name}-%{version}-%{release}
 
-Name:		noobaa-core
-Version:	%{noobaaver}
-Release:	%{revision}%{?dist}
-Summary:	NooBaa RPM
+Name: noobaa-core
+Version:  %{noobaaver}
+Release:  %{revision}%{?dist}
+Summary:  NooBaa RPM
 
-License:	Apache-2.0
-URL:        https://www.noobaa.io/
-Source0:	%{noobaatar}
-Source1:    %{nodetar}
+License:  Apache-2.0
+URL:  https://www.noobaa.io/
+Source0:  %{noobaatar}
 
-BuildRequires: systemd
+BuildRequires:  systemd
+BuildRequires:  python3
+BuildRequires:  make
+BuildRequires:  gcc-c++
+BuildRequires:  boost-devel
 
-Recommends: jemalloc
-
-%global __os_install_post %{nil}
+Recommends:     jemalloc
 
 %global __requires_exclude ^/usr/bin/python$
 
 %description
 NooBaa is a data service for cloud environments, providing S3 object-store interface with flexible tiering, mirroring, and spread placement policies, over any storage resource that allows GET/PUT including S3, GCS, Azure Blob, Filesystems, etc.
 
 %prep
-mkdir noobaa-core-%{version}-%{revision}
-mkdir node-%{nodever}
-tar -xzf %{SOURCE0} -C noobaa-core-%{version}-%{revision}/
-tar -xJf %{SOURCE1} -C node-%{nodever}/
+%setup -n noobaa -q
+
+%build
+NODEJS_VERSION="%{nodever}"
+SKIP_NODE_INSTALL=1 source src/deploy/NVA_build/install_nodejs.sh $NODEJS_VERSION
+
+mkdir -p ../node/
+
+nodepath=$(download_node)
+tar -xJf ${nodepath} -C ../node/
 
-%clean
-[ ${RPM_BUILD_ROOT} != "/" ] && rm -rf ${RPM_BUILD_ROOT}
+PATH=$PATH:%{_builddir}/node/node-v$NODEJS_VERSION-linux-$(get_arch)/bin
+
+npm install --omit=dev && npm cache clean --force
+
+./src/deploy/NVA_build/clone_s3select_submodules.sh
+
+if [[ "%{CENTOS_VER}" = "8" ]]
+then
+  sed -i 's/\/lib64\/libboost_thread.so.1.75.0/\/lib64\/libboost_thread.so.1.66.0/g' ./src/native/s3select/s3select.gyp
+  echo "Using libboost 1.66 for S3 Select"
+fi
+
+GYP_DEFINES="BUILD_S3SELECT=%{BUILD_S3SELECT} BUILD_S3SELECT_PARQUET=%{BUILD_S3SELECT_PARQUET}" npm run build
 
 %install
 rm -rf $RPM_BUILD_ROOT
-mkdir -p $RPM_BUILD_ROOT/usr/local/
 
-cp -R %{_builddir}/%{name}-%{version}-%{revision}/noobaa $RPM_BUILD_ROOT/usr/local/noobaa-core
-cp -R %{_builddir}/node-%{nodever}/* $RPM_BUILD_ROOT/usr/local/noobaa-core/node
+mkdir -p $RPM_BUILD_ROOT/usr/local/
+mv %{_builddir}/noobaa $RPM_BUILD_ROOT/usr/local/noobaa-core
+mv %{_builddir}/node/* $RPM_BUILD_ROOT/usr/local/noobaa-core/node
 
 mkdir -p $RPM_BUILD_ROOT/usr/local/noobaa-core/bin
 ln -s /usr/local/noobaa-core/node/bin/node $RPM_BUILD_ROOT/usr/local/noobaa-core/bin/node
 ln -s /usr/local/noobaa-core/node/bin/npm $RPM_BUILD_ROOT/usr/local/noobaa-core/bin/npm
 ln -s /usr/local/noobaa-core/node/bin/npx $RPM_BUILD_ROOT/usr/local/noobaa-core/bin/npx
 
 mkdir -p $RPM_BUILD_ROOT%{_unitdir}/
-cp -R %{_builddir}/%{name}-%{version}-%{revision}/noobaa/src/deploy/noobaa_nsfs.service $RPM_BUILD_ROOT%{_unitdir}/noobaa_nsfs.service
+mv %{_builddir}/noobaa/src/deploy/noobaa_nsfs.service $RPM_BUILD_ROOT%{_unitdir}/noobaa_nsfs.service
```

Comment:
This should be:

`mv $RPM_BUILD_ROOT/usr/local/noobaa-core/src/deploy/noobaa_nsfs.service $RPM_BUILD_ROOT%{_unitdir}/noobaa_nsfs.service`

Reply:
Addressed.